osu.Framework/Graphics/UserInterface/TabControl.cs

```diff
@@ -71,27 +74,59 @@ public IReadOnlyList<T> Items
         /// </summary>
         public bool AutoSort { set; get; }
 
-        protected Dropdown<T> Dropdown;
+        /// <summary>
+        /// The <see cref="Dropdown"/> which is displayed when tabs overflow the visible bounds of this <see cref="TabControl{T}"/>.
```

Comment:
nit: this is referencing the field and not the generic type, which feels slightly more useful from a documentation perspective

```suggestion
        /// The <see cref="Dropdown{T}"/> which is displayed when tabs overflow the visible bounds of this <see cref="TabControl{T}"/>.
```